apps/docs/pages/guides/database/inspect.mdx

```diff
@@ -0,0 +1,174 @@
+import Layout from '~/layouts/DefaultGuideLayout'
+
+export const meta = {
+  id: 'json',
+  title: 'Inspecting your Postgres database',
+  description:
+    'Inspecting your Postgres database for common issues around disk, query performance, index, locks, and more using the terminal.',
+}
+
+## Inspect from the terminal
+
+The Supabase CLI comes with a range of tools to help inspect your Postgres instances for potential issues. Most tools are compatible with any Postgres databases regardless if they are a Supabase project or not.
+
+### Installing the Supabase CLI
```

Comment:
Let's remove this section and then just link to the installation steps in the CLI Guides